### ISSUE TITLE
Ionic2 Bug - Ios status bar overlap problem

### ISSUE THREAD

**Issue by LeeTaeHyung** [ionic-team/ionic#10689]
**Ionic version:**  (check one with "x")
[ ] **1.x**
[x] **2.x**

**I'm submitting a ...**  (check one with "x")
[x] bug report
[ ] feature request
[ ] support request => Please do not submit support requests here, use one of these channels: https://forum.ionicframework.com/ or http://ionicworldwide.herokuapp.com/

**Current behavior:**
When I go back to the app after calling other apps (Camera, Gallery, Facebook, Google Plus ..) on iOS, there is a problem where the top statusbar overlaps.

**Expected behavior:**
The status bar should not overlap.

**Steps to reproduce:**
Ionic native Camera, when you return to the current app after calling the camera app, the top status bar overlaps.

**Related code:**
Part of the source content
-- StatusBar.styleDefault();
-- StatusBar.overlaysWebView(false);
Call these two methods

**Other information:**
- none

**Ionic info:** (run `ionic info` from a terminal/cmd prompt and paste output below):
Cordova CLI: 6.5.0 
Ionic Framework Version: 2.1.0
Ionic CLI Version: 2.2.1
Ionic App Lib Version: 2.2.0
Ionic App Scripts Version: 1.1.3
ios-deploy version: 1.9.1 
ios-sim version: 5.0.13 
OS: macOS Sierra
Node Version: v6.9.5
Xcode version: Xcode 8.2.1 Build version 8C1002

insert the output from ionic info here
-none

**Comment by jgw96** [comment]
Thanks, we will look into this!

**Comment by drettig** [comment]
Same problem for me. Any solution?

**Comment by andreyliventsev** [comment]
Same problem.

Cordova: 6.4.0
cordova-ios: 4.3.0
xcode: 8.2.1
cordova-plugin-statusbar 2.2.1

config.xml:
<preference name="StatusBarOverlaysWebView" value="false"/>

Initialy status bar is not overlayed on application. 
But after losing focus from any native functionality (camera, contact list) status bar overlaps application. It seems like it has been reinitalized.

Statusbar plugin has js api and after calling 
StatusBar.overlaysWebView(true);
StatusBar.overlaysWebView(false);
it restores statusbar overlap.

**Comment by soramister** [comment]
Any updates on this?

**Comment by LeeTaeHyung** [comment]
Why does not this problem get fixed soon? This is not a small problem. Thank you for your quick patch.

**Comment by yechenchao** [comment]
I have been stuck by this problem for a few weeks, and I finally worked out a solution.

If you have uploaded all your source code to Github or any other providers, pull down a brand new copy from it, then do `npm install` as normal, then the problem was fixed. I am not sure what exactly caused this, but in my case I think this could be cause by some hidden tmp files (`.xxx` files)stored locally which are not ignored to git that affected the process of compiling ionic.
